cloud/jenkins/pxc_operator_aks_latest.groovy

```diff
@@ -199,7 +224,7 @@ void runTest(Integer TEST_ID) {
 
                     export DEBUG_TESTS=1
                     [[ "$CLUSTER_WIDE" == "YES" ]] && export OPERATOR_NS=pxc-operator
-                    [[ "$OPERATOR_IMAGE" ]] && export IMAGE=$OPERATOR_IMAGE || export IMAGE=perconalab/percona-xtradb-cluster-operator:$GIT_BRANCH
+                    export IMAGE=$IMAGE_OPERATOR
```

Comment:
Will not this result in incorrect PARAMS_HASH for test? Please run test with some  `GIT_BRANCH`

Reply:
I ran it here: https://cloud.cd.percona.com/view/PXC/job/pxc-operator-aks-version/90/
It doesn't affect the correct calculation of PARAMS_HASH